### PR TITLE
Updates following Christian's comments

### PR DESCRIPTION
**Pull request by mrdeep1**

### CODE REVIEW

File: draft-ietf-core-new-block.xml [hunk 1]
```diff
@@ -983,9 +965,10 @@ Table 4: Congestion Control Parameters]]></artwork>
         newly derived value for MAX_PAYLOADS should be used for both ends of
         this particular CoAP peer link. Note that the CoAP peer will not know
         about the MAX_PAYLOADS change until it is reconfigured. As a
-        consequence of not being reconfigured, the peer may indicate that
-        there are some missing payloads prior to the actual payload being
-        transmitted as all of its MAX_PAYLOADS payloads have not arrived.</t>
+        consequence of the two peers having different MAX_PAYLOAD values, a
+        peer may continue indicate that there are some missing payloads as
+        all of its MAX_PALOAD set may not have arrived. How the two peer
```
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
        all of its MAX_PAYLOADS set may not have arrived. How the two peer
```

File: draft-ietf-core-new-block.txt [hunk 2]
```diff
@@ -1094,9 +1075,10 @@ Internet-Draft      Quick Block-Wise Transfer Options      February 2021
    newly derived value for MAX_PAYLOADS should be used for both ends of
    this particular CoAP peer link.  Note that the CoAP peer will not
    know about the MAX_PAYLOADS change until it is reconfigured.  As a
-   consequence of not being reconfigured, the peer may indicate that
-   there are some missing payloads prior to the actual payload being
-   transmitted as all of its MAX_PAYLOADS payloads have not arrived.
+   consequence of the two peers having different MAX_PAYLOAD values, a
```
Comment (boucadair) [inline draft-ietf-core-new-block.txt]:
```suggestion
   consequence of the two peers having different MAX_PAYLOADS values, a
```

File: draft-ietf-core-new-block.txt [hunk 3]
```diff
@@ -1094,9 +1075,10 @@ Internet-Draft      Quick Block-Wise Transfer Options      February 2021
...
+   peer may continue indicate that there are some missing payloads as
+   all of its MAX_PALOAD set may not have arrived.  How the two peer
```
Comment (boucadair) [inline draft-ietf-core-new-block.txt]:
```suggestion
   all of its MAX_PAYLOADS set may not have arrived.  How the two peer
```

File: draft-ietf-core-new-block.xml (hunk 1 above)
Comment (boucadair) [inline draft-ietf-core-new-block.xml]:
```suggestion
        all of its MAX_PAYLOADS set may not have arrived. How the two peer
```

**Review by boucadair — CHANGES_REQUESTED** [review]

File: draft-ietf-core-new-block.xml (hunk 1 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Fixed

File: draft-ietf-core-new-block.txt (hunk 2 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.txt]: Fixed

File: draft-ietf-core-new-block.txt (hunk 3 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.txt]: Fixed

File: draft-ietf-core-new-block.xml (hunk 1 above)
Comment (mrdeep1) [reply draft-ietf-core-new-block.xml]: Fixes

**Comment by mrdeep1** [comment]
Updated code pushed

**Pull request merged**